### PR TITLE
demo ascii art of socks

### PR DESCRIPTION
**Pull request by idt911-jmiller**
@Oshuary what are your **thoughts**?
### Socks
- [x] are fancy?
- [x] aren't fancy enough

### CODE REVIEW

**Comment by macdaddyaz** [comment]
I pass judgement before seeing them.

**Comment by Oshuary** [comment]
Absolutely agree @idt911-jmiller

**Comment by idt911-jmiller** [comment]
#20 has conflicts

File: fancy.MD [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Super fancy
+
+           mmm          mmm          mmm          mmm
+      mmm  | |     mmm  | |     mmm  | |     mmm  | |
+      | |  | |     | |  | |     | |  | |     | |  | |
```
Comment (Oshuary) [inline fancy.MD]: @idt911-jmiller  Do you mind if I make them more fancy at some point?

File: fancy.MD [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Super fancy
```
Comment (cody-lny) [inline fancy.MD:1]: Wait... **HOW** fancy are these socks?

File: fancy.MD [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+           mmm          mmm          mmm          mmm
+      mmm  | |     mmm  | |     mmm  | |     mmm  | |
+      | |  | |     | |  | |     | |  | |     | |  | |
+      | |  \) \    | |  \) \    | |  \) \    | |  \) \
+      \) \  \__)   \) \  \__)   \) \  \__)   \) \  \__)
```
Comment (Oshuary) [inline fancy.MD:7]: @idt911-jmiller  Do you mind if I make them more fancy in some commit down the road?

File: fancy.MD (hunk 2 above)
Comment (idt911-jmiller) [inline fancy.MD:1]: Very.

**Comment by idt911-jmiller** [comment]
@Oshuary stripes completed.

**Pull request merged**